vitals/batteries/Battery.java

```diff
@@ -0,0 +1,16 @@
+package vitals.batteries;
+
+public class Battery {
+	public int temperature=0;
+	public int maxTemperature=0;
```

Comment:
Do you need a class member when there is a constant already defined below? To me it seems redundant as the max and min values are fixed for the program lifetime. Please check.